overrides/main.html

```diff
@@ -7,7 +7,6 @@
 
 {% block announce %}
 <p>
-  We are phasing out use of the `renovate` npm user, which affects about 10 early users of the Renovate App on github.com.
-  Read <a href="https://github.com/renovatebot/renovate/discussions/20613" target="_blank">discussion 20613 on the Renovate repository</a> to learn more.
+  We released <code>v35</code> of Renovate. Read the <a href="https://docs.renovatebot.com/release-notes-for-major-versions/" target="_blank">Release notes for major versions of Renovate</a> section of the docs to learn what's changed.
```

Comment:
![announcement-bar-v35](https://user-images.githubusercontent.com/34918129/224375312-5d13b707-214e-494a-95df-3342f869a1c5.png)

Preview of the announcement bar.